components/resources/brave_vpn_strings.grdp

```diff
@@ -258,4 +258,10 @@ However, we noticed this helper service is not running - and we don't want to le
   <message name="IDS_BRAVE_VPN_SUPPORT_TIMEZONE" desc="Label on dialog to report bug to technical support ">
     Timezone:
   </message>
+  <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_TITLE" desc="Title text for session expired panel">
+    Session expired
+  </message>
+  <message name="IDS_BRAVE_VPN_MAIN_PANEL_SESSION_EXPIRED_PART_CONTENT" desc="Content text for session expired panel">
+    Please login to your <ph name="BRAVE_ACCOUNT_BEGIN">$1</ph>Brave Account<ph name="BRAVE_ACCOUNT_END">$2</ph> to refresh your VPN session.
```

Comment:
Nit: should be s/login/log in

/cc @rmcfadden3

Reply:
Pushed https://github.com/brave/brave-core/pull/17576 :)

Reply:
Yeah, confirmed: this should say `log in` as two words.